modules/installation-guide/partials/proc_using-minikube-to-set-up-kubernetes.adoc

```diff
@@ -17,5 +17,7 @@ This section describes how to use Minikube to set up a local single-node {kubern
 . Start Minikube (it is important to *allocate at least 4 GB of RAM*):
 +
 ----
-$ minikube start --memory=4096
+$ minikube start --addons=ingress --memory=4096
```

Comment:
On linux isn't `--vm=true` the default?
If that's the case we could start using the same command on all platforms:
```
minikube start --addons=ingress --memory=4096 --vm=true
```

Reply:
Another doubt: is 4G enough to run Che multi-user, start Quarkus stack, compile and run it? I am afraid that this is not enough anymore. If that's the case I would change to "at least 4GB of RAM but 8GB are recommended" and use `--memory=8192` in the command line.

Reply:
On Linux option `--wm=true` is not required. And from resource usage docker driver behaves better.
But if you think that we should include `--wm=true`  as default, I may add it.

Reply:
Agree, the memory amount is close very minimal.